SU-SWS/stanford_basic
# D8CORE-4037: fixed word wrapping with very long menu items

# READY FOR REVIEW

# Summary
- Quick change to correct problems with some (but not all) particularly long menu names failing to wrap correctly in dropdown menus

# Review By (Date)
- end of sprint
- 
# Urgency
- low
- 
# Steps to Test

1. Checkout branch, clear caches, create dropdown menus with long items, ensure they wrap in some kind of sane way.
2. pics attached.

# Affected Projects or Products
- Does this PR impact any particular projects, products, or modules?

# Associated Issues and/or People
- JIRA ticket
- Other PRs
- Any other contextual information that might be helpful (e.g., description of a bug that this PR fixes, new functionality that it adds, etc.)
- Anyone who should be notified? (`@mention` them here)

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
